packages/fullstack/src/render.rs

```diff
@@ -50,6 +51,11 @@ where
     }
 }
 
+pub enum SSRError {
+    Incremental(IncrementalRendererError),
+    Routing(ParseRouteError),
+}
+
```

Comment:
I believe these need docs if they're part of our public API?

Reply:
It should be private because of the export. I marked it as pub(crate) to make it more clear